pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   # webview plugin
-  flutter_inappwebview: 6.0.0
+  flutter_inappwebview: 6.0.0-beta.22
```

Comment:
It is actually being used on the `cn_supported` branch. Please use `cn_supported` branch